apps/api-frontend/src/app/app.tsx

```diff
@@ -23,6 +23,15 @@ export const App = () => {
           </Switch>
         </Router>
       </main>
+      <div className="footer">
```

Comment:
looks like this footer doesn't move to the bottom of the page in the results page and instead overlaps with other text.